src/main/groovy/com/morpheusdata/digitalocean/DigitalOceanOptionSourceProvider.groovy

```diff
@@ -129,14 +129,18 @@ class DigitalOceanOptionSourceProvider implements OptionSourceProvider {
 		// if we know the cloud then load from cached data
 		if(cloudId) {
 			cloud = morpheus.services.cloud.get(cloudId)
-			morpheus.services.referenceData.list(new DataQuery().withFilter("category", "digitalocean.${cloudId}.vpc")).each { ReferenceDataSyncProjection refData ->
+			morpheus.services.referenceData.list(new DataQuery().withFilter("category", "digitalocean.${cloudId}.vpc"))
```

Comment:
This should be searching VPCs through morpheus cloud pools with something like  `morpheus.services.cloudPool.list(new DataQuery().withFilter("cloud.id", cloudId).withFilter("regionCode", datacenter)).each { ... }`

The `each {}` block will need to be adapted to the cloud pools data.

Reply:
@ddeyoung Code commited as suggested. Please review.